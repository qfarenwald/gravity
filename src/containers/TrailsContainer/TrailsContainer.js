import React from 'react';
import { Trail } from '../Trail/Trail';
import { connect } from 'react-redux';
import './TrailsContainer.css';

const TrailsContainer = ({ trails }) => {
  console.log('trailsFromState', trails)
  // const trailsToDisplay = trails.map((trail) => {
  //
  // })
  return (
    <div>
      <Trail />
    </div>
  )
}

export const mapStateToProps = state => ({
  trails: state.trails
});

export default connect(mapStateToProps)(TrailsContainer)
