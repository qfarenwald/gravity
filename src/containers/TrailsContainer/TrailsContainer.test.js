import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { TrailsContainer } from './TrailsContainer';

describe('TrailsContainer', () => {
  const mockMatch = {
    location: {
      pathname: "/hello/hey/hi"
    }
  }

  it('should match snapshot', () => {
    const wrapper = shallow(<TrailsContainer match={mockMatch} trails={[
      {
        key: 46286,
        id: 46286,
        name: 'Betasso Preserve',
        stars: 4,
        difficulty: 'greenBlue',
        descent: -829,
        conditionDate: '2019-10-03 22:57:25',
        conditionStatus: 'All Clear',
      },
      {
        key: 601365,
        id: 601365,
        name: 'Walker Ranch',
        stars: 3.9,
        difficulty: 'blueBlack',
        descent: -1510,
        conditionDate: '2019-09-19 07:48:20',
        conditionStatus: 'All Clear',
      },
    ]}
    />);

    expect(wrapper).toMatchSnapshot();
  });
});
