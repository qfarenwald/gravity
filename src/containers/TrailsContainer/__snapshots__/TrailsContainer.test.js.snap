// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrailsContainer should match snapshot 1`] = `
<div
  className="TrailsContainer"
>
  <h3>
    PICK YOUR TRAIL
  </h3>
  <Trail
    conditionDate="2019-10-03 22:57:25"
    conditionStatus="All Clear"
    descent={-829}
    difficulty="greenBlue"
    id={46286}
    key="46286"
    name="Betasso Preserve"
    stars={4}
  />
  <Trail
    conditionDate="2019-09-19 07:48:20"
    conditionStatus="All Clear"
    descent={-1510}
    difficulty="blueBlack"
    id={601365}
    key="601365"
    name="Walker Ranch"
    stars={3.9}
  />
  <Link
    className="link"
    to="/"
  >
    <h5>
      BACK TO LOCATIONS
    </h5>
  </Link>
</div>
`;
